### PR TITLE
Final draft for the 1.15 Release Notes

### PR DESCRIPTION
**Pull request by onyiny-ang**
The PR is based off of this [HackMD](https://hackmd.io/V70KJ4nmQpeq7DIHlDw8Ow) draft.
This must be merged before the big red button is pushed to start building the 1.15 release :) 

Thanks to the 1.15 release team for all their hard work in making this beautiful _and_ readable! :heart_eyes_cat: 

/assign @claurence 
/cc @jeefy  @saschagrunert @nickchase @alokpatra

### CODE REVIEW

**Comment by saschagrunert** [comment]
LGTM, thanks! :+1:

File: releases/release-1.15/release-notes-draft.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## 1.15 What’s New
 
-A complete changelog for the release notes is now hosted in a customizable format at [https://relnotes.k8s.io/](https://relnotes.k8s.io/). Check it out and please give us your feedback!
+A complete changelog for the release notes is now hosted in a customizable format at [https://relnotes.k8s.io/](https://relnotes.k8s.io/?releaseVersions=1.15.0). Check it out and please give us your feedback!
```
Comment (saschagrunert) [inline releases/release-1.15/release-notes-draft.md]:
```suggestion
A complete changelog for the release notes is now hosted in a customizable format at [https://relnotes.k8s.io/](https://relnotes.k8s.io/?releaseVersions=1.15). Check it out and please give us your feedback!
```

File: releases/release-1.15/release-notes-draft.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ These changes are reflected in the following Kubernetes enhancements:
 - ExecutionHook API [designed to trigger hook commands](https://github.com/kubernetes/enhancements/issues/962) in containers is now Alpha.
 - Continued deprecation of extensions/v1beta1, apps/v1beta1, and apps/v1beta2 APIs; these extensions will be retired in 1.16!
 
-Check the [release notes website](https://relnotes.k8s.io/) for the complete changelog of notable features and fixes.
+Check the [release notes website](https://relnotes.k8s.io/?releaseVersions=1.15.0) for the complete changelog of notable features and fixes.
```
Comment (saschagrunert) [inline releases/release-1.15/release-notes-draft.md]: Check the [release notes website](https://relnotes.k8s.io/?releaseVersions=1.15) for the complete changelog of notable features and fixes.

Comment (saschagrunert) [reply releases/release-1.15/release-notes-draft.md]: I see, compared with https://github.com/kubernetes-sigs/release-notes/pull/63 this makes totally sense. Sorry for the interruption

File: releases/release-1.15/release-notes-draft.md (hunk 1 above)
Comment (saschagrunert) [reply releases/release-1.15/release-notes-draft.md]: I see, compared with https://github.com/kubernetes-sigs/release-notes/pull/63 this makes totally sense. Sorry for the interruption

**Comment by saschagrunert** [comment]
LGTM again.

File: releases/release-1.15/release-notes-draft.md [hunk 3]
```diff
@@ -1,668 +1,553 @@
...
-### SIG Apps, SIG Instrumentation, and SIG Scalability
-
-- The `series.state` field in the events.k8s.io/v1beta1 Event API is deprecated and will be removed in v1.18 ([#75987](https://github.com/kubernetes/kubernetes/pull/75987), [@yastij](https://github.com/yastij))
-
-### SIG Cloud Provider, and SIG VMWare
+A complete changelog for the release notes is now hosted in a customizable format at [https://relnotes.k8s.io/](https://relnotes.k8s.io/). Check it out and please give us your feedback!
 
-- Fix vSphere SAML token auth when using Zones ([#78137](https://github.com/kubernetes/kubernetes/pull/78137), [@dougm](https://github.com/dougm))
-- Support vSphere SAML token auth when using Zones ([#75515](https://github.com/kubernetes/kubernetes/pull/75515), [@dougm](https://github.com/dougm))
+Kubernetes 1.15 consists of **26 enhancements**: 3 moving to stable, 13 in beta, and 10 in alpha. The main themes of this release are:
```
Comment (claurence) [inline releases/release-1.15/release-notes-draft.md]: update - 2 moving to stable 
Admissions webhook is staying as beta but adding additional capabilities

Comment (onyiny-ang) [reply releases/release-1.15/release-notes-draft.md]: Thanks @claurence. Updating now. Have also pinged @castrojo to update this for the blog post.

**Comment by claurence** [comment]
Thanks Lindsey and team for the work here 

/lgtm 
/approve
/hold 

feel free to remove hold when y'all are ready to go live

**Comment by onyiny-ang** [comment]
Thanks @claurence  :)
/hold cancel

**Pull request merged**